index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+      integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <h5>Ismail Changezi</h5>
+        <i class="fa-solid fa-bars"></i>
+      </nav>
+      <section>
+        <h4>Hey There.<br />I'm Ismail</h4>
```

Comment:
- [ ] [OPTIONAL] _(This is marked as optional because the last reviewer did not mention it. But it is highly recommended that you implement it.)_  Consider replacing this `h4 ` with `h1` because of SEO reasons. [Click here to learn more about this
](https://www.contentkingapp.com/academy/headings/#:~:text=A%20page%20starts%20with%20the,even%20add%20H4%2DH6%20subheadings%20.&text=A%20page%20with%20H1%2C%20H2,a%20level%20in%20the%20hierarchy.)